### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
Added DisableFtpParsing

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3503

This is avaialble only in windows 11 and windows 2022

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -507,6 +508,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableFtpParsing
+
+Specifies whether to disable FTP Parsing for Network Protection.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:513]:
```suggestion
Specifies whether to disable FTP parsing for network protection.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**